Appraisals

```diff
@@ -51,7 +51,7 @@ appraisals = {
   },
   no_rails: proc {},
   rspec_lt_3_10: proc { |with_rails|
-    version = "< 3.10"
+    version = "~> 3.9"
```

Comment:
@mcmire This needs to be `version = "~> 3.9.0" if you want to ensure versions less than 3.10 are loaded.  With `version = "~> 3.9"` it will just load the current 3.12 version.

Reply:
D'oh! Thanks. Fixed in [21a4bcf](https://github.com/mcmire/super_diff/pull/179/commits/21a4bcf7fbf4352da95f7e299b9a80bbc9c7b191).